javascript_nodejs/09.message-routing/index.js

```diff
@@ -79,10 +79,6 @@ const memoryStorage = new MemoryStorage();
 const conversationState = new ConversationState(memoryStorage);
 const userState = new UserState(memoryStorage);
 
-// Use the BotStateSet middleware to automatically read and write conversation and user state.
```

Comment:
You might want to incldue onTurnError handler. See the nlp-with-dispatch bot for the correct code.

Reply:
I'd also move it into the samples folder.  I'll merge it once those two things are addressed

Reply:
Thanks for reviewing! Updating the PR in a few minutes :)